src/main/java/de/tum/cit/aet/artemis/core/web/open/PublicAccountResource.java

```diff
@@ -166,13 +169,16 @@ public ResponseEntity<UserDTO> getAccount() {
         }
 
         User user = userOptional.get();
-        boolean hasUserRegisteredAPasskey = this.passkeyCredentialsRepository.existsByUserId(user.getId());
+        boolean askToSetupPasskey = false;
+        if (askUsersToSetupPasskey) {
+            askToSetupPasskey = !this.passkeyCredentialsRepository.existsByUserId(user.getId());
+        }
```

Comment:
rather a nitpick, but these two variable names are very similar, maybe the local one can be changed into something like this? `shouldPromptUserToSetupPasskey`

Reply:
I do aggree with your point, I though of something similar but decided against it because I thought it could be confusing aswell